test/testinput/bufr_adpsfc.yaml

```diff
@@ -0,0 +1,190 @@
+
+          pressureMeanSeaLevel:
+            mnemonic: PMSL
+
+          stationElevation:
+            mnemonic: SELV
+
+          temperatureAir:
+            mnemonic: TMDB
+
+          temperatureDewpoint:
+            mnemonic: TMDP
+        
+#        temperatureSeaSurface:
+#          mnemonic: SST1
+
+          windDirection:
+            mnemonic: WDIR 
+
+          windSpeed:
+            mnemonic: WSPD
+
+          hourofreceipt:
+            mnemonic: RCHR
+
+          minuteofreceipt:
+            mnemonic: RCMI
+
+    ioda:
+      backend: netcdf 
+      obsdataout: "./testoutput/gdas.t06z.adpsfc.tm00.nc"
+
+      dimensions:
+        - name: "nlocs"
+          size: variables/latitude.nrows #implicit info
+
+#        - name: "nme: "datetime@MetaData"
+#        #          source: "timestamp"
+#        #          dimensions: [ "nlocs" ]
+#        #          longName: "Datetime"
+#        #          units: "datetime"chans"
+#          size: "{LENGTH}" #explicit info
+
+      variables:
+        - name: "datetime@MetaData"
+          coordinates: "longitude latitude"
+          source: variables/timestamp
+          dimensions: [ "nlocs" ]
+          longName: "Datetime"
+          units: "datetime"
+
+        - name: "WMOBlock@MetaData"
+          source: variables/WMOBlock
+          dimensions: [ "nlocs" ]
+          longName: "WMO_BLOCK_NUMBER"
+          units: ""
+        
+        - name: "WMOStation@MetaData"
+          source: variables/WMOStation
+          dimensions: [ "nlocs" ]
+          longName: "WMO_STATION_NUMBER"
+          units: ""
+
+        - name: "latitude@MetaData"
+          source: variables/latitude
+          dimensions: ["nlocs"]
+          longName: "LATITUDE (COARSE ACCURACY)"
+          units: "degrees_north"
+          range: [-90, 90]
+
+        - name: "longitude@MetaData"
+          source: variables/longitude
+          dimensions: ["nlocs"]
+          longName: "LONGITUDE (COARSE ACCURACY)"
+          units: "degrees_east"
+          range: [-180, 180]
+
+        - name: "stationElevation@MetaData"
+          coordinates: "longitude latitude"
+          source: variables/stationElevation
+          dimensions: ["nlocs"]
+          longName: "HEIGHT OF STATION"
+          units: "Meter"
+#          range: [0, 10000]
+
+        - name: "pressureAir@ObsValue"
+          coordinates: "longitude latitude"
+          source: variables/pressureAir
+          dimensions: ["nlocs"]
+          longName: "PRESSURE"
+          units: "Pa"
+
+        - name: "pressureMeanSeaLevel@ObsValue"
+          coordinates: "longitude latitude"
+          source: variables/pressureMeanSeaLevel
+          dimensions: ["nlocs"]
+          longName: "PRESSURE REDUCED TO MSL"
+          units: "Pa"
+
+        - name: "temperatureAir@ObsValue"
+          source: variables/temperatureAir
+          dimensions: ["nlocs"]
+          longName: "TEMPERATURE/DRY BULB TEMPERATURE"
+          units: "K"
```

Comment:
Consider adding "range: " to this parameter and other ones where this might apply.